apollo-router/src/logging/mod.rs

```diff
@@ -84,7 +84,14 @@ macro_rules! assert_snapshot_subscriber {
         $crate::logging::test::SnapshotSubscriber::create_subscriber(|yaml| {
             insta::with_settings!({sort_maps => true}, {
                 // the tests here will force maps to sort
-                insta::assert_yaml_snapshot!(yaml);
+                let mut settings = insta::Settings::clone_current();
```

Comment:
could you explain the goal of this change and the snapshot- files renaming? It does not look related to the JWKS parsing issue

Reply:
It's because the experience of using the logging tests that also uses insta wasn't great as you end up with snapshots suffixed with an index number. It only manifested itself in this issue as I had not used this pattern before, but can pull out into a separate PR if you like.

Reply:
it's ok, let's merge it